pkg/util/strings.go

```diff
@@ -19,3 +19,30 @@ func UniqueStrings(strings []string) []string {
 	sort.Strings(strings)
 	return strings
 }
+
+// IsWildcardMatch matches the given input string against the provided pattern. The
+// pattern might contain '?' and '*' wildcards.
+// Original code: https://discuss.leetcode.com/topic/9350/python-dp-solution/5
+func IsWildcardMatch(s string, p string) bool {
```

Comment:
done... also the other function in this file is not used anywhere, so will just nuke this